src/startStopScene.ts

```diff
@@ -71,7 +71,7 @@ export async function startScene(lineStart?: number) {
 
     // Create the command
     const filePath = editor.document.fileName;  // absolute path
-    const cmds = ["manimgl", filePath, sceneName];
+    const cmds = ["manimgl", `\"${filePath}\"`, sceneName];
```

Comment:
You've escaped the `"` here, but do we have to? For me, it works fine even without the escape, doesn't it for you?

Reply:
for me it works both ways

Reply:
Nice, so if it works both ways, I'd go for the simpler `"`, not `\"`. Maybe let's wait until @RickLuiken confirms that both options work on their machine as well.

Reply:
I believe it's not required with the template string, I just did it out of habit. I can remove them if you want!

Reply:
>  I can remove them if you want!

Yes, that'd be great